### ISSUE TITLE
Extend Pseudo localization en-XA with extra characters 

### ISSUE THREAD

**Issue by asvny** [formatjs/formatjs#2698]
## Which package?
formatjs-cli

## Describe the bug
Currently the formatjs cli does pseudo localization by replacing the characters with different accents but it would be much better if it can add some extra characters in it.
Example
```
For a text like this - Message with a @I18N@value@I18N@ placeholder 

Current 
 Ḿèśśâĝè ẘíṭḫ â @Ḭ18Ŋ@ṿâĺůè@Ḭ18Ŋ@ ṗĺâćèḫŏĺḋèŕ

Expected
Ṁéšššàĝĝé ŵîîţĥ àà @Î18Ñ@ññôñ-ššîñĝĝļé-ŵŵôŕðð@Î18Ñ@ þþļàççéĥôôļðééŕ
```

## Additional context
This will be useful for testing like visual layout breaks, word break/wrap etc...,

Happy to send a PR ..if you're okay with this.

**Comment by longlho** [comment]
I'd be open to this under a different pseudo-locale.

**Comment by asvny** [comment]
Thats cool .. it will be a non breaking change .. what do you want to call that pseudo locale ?

**Comment by longlho** [comment]
Hmm I don't have a strong opinion on what it should be called as long as it's not a real country code

**Comment by longlho** [comment]
fixed by #2708
